### PR TITLE
Add host

### PR DESCRIPTION
**Pull request by kapouer**

### CODE REVIEW

**Comment by hayd** [comment]
This is osx too, so I think it should be in common.

**Comment by kapouer** [comment]
Sure, though i'm not sure if osx args are the same (?)

**Comment by hayd** [comment]
This is osx:

```
 % host
Usage: host [-aCdlriTwv] [-c class] [-N ndots] [-t type] [-W time]
            [-R number] [-m flag] hostname [server]
       -a is equivalent to -v -t ANY
       -c specifies query class for non-IN data
       -C compares SOA records on authoritative nameservers
       -d is equivalent to -v
       -l lists all hosts in a domain, using AXFR
       -i IP6.INT reverse lookups
       -N changes the number of dots allowed before root lookup is done
       -r disables recursive processing
       -R specifies number of retries for UDP packets
       -s a SERVFAIL response should stop query
       -t specifies the query type
       -T enables TCP/IP mode
       -v enables verbose output
       -w specifies to wait forever for a reply
       -W specifies how long to wait for a reply
       -4 use IPv4 query transport only
       -6 use IPv6 query transport only
       -m set memory debugging flag (trace|record|usage)
```
Specifically, your TLDR all seem to work :)

**Comment by kapouer** [comment]
Moved to common

File: pages/common/host.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# host
+
+Lookup Domain Name Server
+
+- Lookup A, AAAA, and MX records of a domain
+
+`host github.com`
```
Comment (igorshubovych) [inline pages/common/host.md]: would not it be better to use {{domain}} here?

File: pages/common/host.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Lookup CNAME, TXT,... of a domain
+
+`host -t TXT github.com`
+
+- Reverse lookup an IP
+
+`host 8.8.8.8`
```
Comment (igorshubovych) [inline pages/common/host.md]: How about {{ip4}} instead of 8.8.8.8?

Comment (kapouer) [inline pages/common/host.md]: Sure ! I did not know how to write it properly, will update PR in a moment

**Comment by kapouer** [comment]
@igorshubovych i overwrote the patch to include your suggestions. Thank you !

File: pages/common/host.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Lookup Domain Name Server
+
+- Lookup A, AAAA, and MX records of a domain
+
+`host {{domain}}`
+
+- Lookup CNAME, TXT,... of a domain
+
+`host -t TXT {{domain}}`
```
Comment (ostera) [inline pages/common/host.md]: I'd use {{field}} instead of TXT

**Comment by kapouer** [comment]
@ostera indeed, i also took the liberty to rephrase it a little

**Comment by ostera** [comment]
Looks good to me.

**Comment by waldyrious** [comment]
I'm not too comfortable with the `{{ip4}}`, sorry for not commenting earlier. I would say something that better resembles an IP address (but clearly indicates it is a sample) would work better, e.g. `{{123.123.123.123}}`. Thoughts @igorshubovych @kapouer?

**Comment by kapouer** [comment]
Well, https://github.com/tldr-pages/tldr/search?utf8=✓&q=ip shows the usual wording is
{{ip_address}}

**Comment by kapouer** [comment]
Changed. Especially since an ipv6 address works too.

**Comment by waldyrious** [comment]
That was a better solution, good call. Seems like we have a keeper :)

**Pull request merged**